### PR TITLE
Refactor: use cuex engine

### PR DESCRIPTION
**Pull request by FogDong**
### Description of your changes

copilot:all

Fixes #

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: references/cli/debug.go [hunk 1]
```diff
@@ -380,11 +380,12 @@ func (d *debugOpts) separateBySteps(v cue.Value, ioStreams cmdutil.IOStreams) er
 }
 
 type renderOptions struct {
-	hideIndex    bool
-	filterFields []string
+	// hideIndex    bool
```
Comment (anoop2811) [inline references/cli/debug.go:383]: commented code

Comment (FogDong) [reply references/cli/debug.go:383]: Yeah sorry I want to update the `debug` tool at last, first to make all tests pass.

Comment (anoop2811) [reply references/cli/debug.go:383]: :) Looking forward to merging this once the tests are passing

**Review by anoop2811 — APPROVED** [review]
LGTM. Thank you for the feature and the huge effort!

**Review by wonderflow — APPROVED** [review]
amazing, great work!

**Pull request merged**